minsk-python-meetup/videos/asgi-ne-eshche-odin-gateway-sergei-khaletskii-epam-python-meetup-28-11-2019.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
I think it will be nice to add to each file flag `minimal_download: true` to show that these files require some additional work.

Reply:
`minimal_download: true` in a pyvideo data json file will make the test to fail (it's out of the [schema](https://github.com/pyvideo/data/blob/master/.schemas/video.json))

If you want a "quick draft" of the events you could use `pyvideo_scrape` with [`minimal_download: true`](https://github.com/pyvideo/pyvideo_scrape#use-cases) in it `events.yml`